src/server/data-access/MwBotWikibaseMessagesRepo.ts

```diff
@@ -0,0 +1,75 @@
+import TechnicalProblem from '@/common/data-access/error/TechnicalProblem';
+import MessageNotFound from '@/common/data-access/error/MessageNotFound';
+import MessageTranslationCollection from '@/datamodel/MessageTranslationCollection';
+import MessageCollection from '@/datamodel/MessageCollection';
+import MessagesRepository from '@/common/data-access/MessagesRepository';
+import mwbot from 'mwbot';
+
+interface RespondMessage {
+	[ key: string]: string;
```

Comment:
It seems that the individual messages returned from `query=allmessages` always have a `name`, `normalizedname`, `*`, and possibly `missing` property, so why not include that in the interface? :)

Also, I tripped over the name for a moment. This is an object that's part of the `allmessages` response, so how about something like `AllMessagesResponseMessage`? <sup>yes, not a great name either</sup>

Reply:
done